voorhoede/head-start
# chore: watch GraphQL files and (remote) translation records

# Changes

- Adds a watcher for GraphQL files and automatically runs `prep:datocms-types` on changes.
- Adds a watcher for translation records in DatoCMS (remote) and automatically runs `prep:download-translations` on changes.

# Associated issue

N/A (just improves DX)

# How to test

1. Checkout this branch locally
2. Run `npm run dev`
3. Make a change to any `.graphql` file
4. Verify types are automatically updated
5. Make a change to a translation record in DatoCMS
6. Verify the updated translations are automatically downloaded

# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have made changes to the README and if the change affects the project setup (npm commands changed, new service added, environmental variable added)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
